crates/aptos-rest-client/src/lib.rs

```diff
@@ -64,13 +66,64 @@ const X_APTOS_SDK_HEADER_VALUE: &str = concat!("aptos-rust-sdk/", env!("CARGO_PK
 
 type AptosResult<T> = Result<T, RestError>;
 
+#[derive(Deserialize)]
+pub struct Table {
+    pub handle: AccountAddress,
+}
+
+#[derive(Deserialize)]
+pub struct OriginatingResource {
```

Comment:
Is this reflecting the `OriginatingAddress` table from `account.move`? If so, why is it called `OriginatingResource` instead of `OriginatingAddress`?

Reply:
u right, i just ported code over from key_rotation.rs